client/post-editor/editor-post-type/index.jsx

```diff
@@ -61,7 +62,8 @@ EditorPostType.propTypes = {
 	siteId: PropTypes.number,
 	isNew: PropTypes.bool,
 	typeSlug: PropTypes.string,
-	type: PropTypes.object
+	type: PropTypes.object,
+	globalId: PropTypes.number
```

Comment:
Looks like this introduced warnings, since `globalId` is a string, not a number:

![image](https://cloud.githubusercontent.com/assets/1779930/19696335/b1ebe50e-9ab4-11e6-91b0-e9ea638828a4.png)